doc/source/user_guide/general_settings.rst

```diff
@@ -23,27 +31,52 @@ failures. THe level of information shown depends on the setting specified for
 
 Reporting mesh quality
 ----------------------
-The ``mesh.quality`` TUI displays information about the quality of the mesh in the
+The following example shows a comparison between the TUI command and the
+python code for displaying information about the quality of the mesh in the
 console, including the minimum orthogonal quality and maximum aspect ratio.
-The level of information that is shown depends on the setting specified for
-``mesh.check-verbosity``.
+
+TUI command
+
+.. code:: scheme
+
+    /mesh/quality
+
+Python command
 
 .. code:: python
 
     session.solver.tui.mesh.quality()
 
 Scaling mesh
 ------------
-The ``mesh.scale`` TUI prompts for the scaling factors in each of the active Cartesian
+The following example shows a comparison between the TUI command and the
+python code for the scaling factors in each of the active Cartesian
 coordinate directions.
 
+TUI command
+
+.. code:: scheme
+
+    /mesh/scale 1 1 1
+
+Python command
+
 .. code:: python
 
     session.solver.tui.mesh.scale(1,1,1)
 
 Defining units
 --------------
-The ``define.units`` TUI sets unit conversion factors.
+The following example shows a comparison between the TUI command and the
+python code for the setting unit conversion factors.
```

Comment:
Looking at the examples I wonder: Does it set conversion factors or does it set the solver units?

Reply:
Either way, maybe it needs some rephrasing? Do we intend: "for the setting" (as written) or "for setting the"?

Reply:
Yes, it does need rephrasing.